fleet/fleet.go

```diff
@@ -234,6 +232,13 @@ func (f fleet) GetStatus(name string) (UnitStatus, error) {
 }
 
 func (f fleet) GetStatusWithExpression(exp *regexp.Regexp) ([]UnitStatus, error) {
+	status, err := f.GetStatusWithMatcher(exp.MatchString)
+	return status, maskAny(err)
+}
+
+// GetStatusWithMatcher returns a UnitStatus for each unit where the given matcher
```

Comment:
Well, not misleading, maybe it's just me - I'd go with `GetStatusWithMatcher returns a list of UnitStatus for each unit where the given matcher returns true`